### DISCUSSION TITLE
Why do the plugin examples use `fs` over Bun's File I/O API?

### DISCUSSION THREAD

**Discussion by ghost** [oven-sh/bun#6490] [Q&A]
I'm writing some internal Bun plugins, and I'm using Bun's examples as a starting point. I noticed that in both the YAML & Svelte plugin examples provided in [Runtime > Plugins](https://github.com/oven-sh/bun/blob/079476729106b246b9d7e4aa14257c58d99901d3/docs/runtime/plugins.md), `const { readFileSync } = await import("fs");` is being used. I was thinking to use `Bun.file` to read in the file, but I wasn't sure if I was missing a reason not to use it 🤔

**Comment by masterbater** [comment]
Compat
